### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-autobind` -> `dom-bind`
`version++`
`polymer++`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
   "homepage": "https://github.com/PolymerElements/iron-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.7"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1"
   },
   "devDependencies": {
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.6",
-    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.9.6",
```
Comment (cdata) [inline bower.json]: There is no `0.9.6` (yet (again)).

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "iron-input",
-  "version": "0.8.0",
+  "version": "0.9.0",
   "authors": "The Polymer Authors",
```
Comment (cdata) [inline bower.json]: Usually this is an array. Who doesn't like arrays?

**Comment by notwaldorf** [comment]
Done!

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**